webapp/channels/src/packages/mattermost-redux/src/actions/users.ts

```diff
@@ -972,13 +972,16 @@ export function updateMe(user: Partial<UserProfile>): ActionFuncAsync<UserProfil
 
 export function saveCustomProfileAttribute(userID: string, attributeID: string, attributeValue: string): ActionFuncAsync {
```

Comment:
There's nothing functionally off here, but the `data` returned here is typed as `any`. Adding the following keeps the typings in check.

```suggestion
export function saveCustomProfileAttribute(userID: string, attributeID: string, attributeValue: string): ActionFuncAsync<Record<string, string>> {
```


Then `const data = ...` instead of `let data;`
```ts
export function saveCustomProfileAttribute(userID: string, attributeID: string, attributeValue: string): ActionFuncAsync<Record<string, string>> {
    return async (dispatch) => {
        try {
            const values = {[attributeID]: attributeValue.trim()};
            const data = await Client4.updateCustomProfileAttributeValues(values);
            return {data};
        } catch (error) {
            dispatch(logError(error));
            return {error};
        }
    };
}

```